shared/menubar/remote-container.desktop.tsx

```diff
@@ -1,89 +1,93 @@
+    },
+    onHideDiskSpaceBanner: () => dispatch(FsGen.createShowHideDiskSpaceBanner({show: false})),
+    onRekey: () => {
+      dispatch(createOpenRekeyPopup())
+      hideWindow()
+    },
+    openApp: (tab?: Tabs.AppTab) => {
+      dispatch(ConfigGen.createShowMain())
+      tab && dispatch(RouteTreeGen.createSwitchTab({tab}))
+    },
+    quit: () => {
+      if (!__DEV__) {
+        if (isLinux) {
+          dispatch(SettingsGen.createStop({exitCode: RPCTypes.ExitCode.ok}))
+        } else {
+          dispatch(ConfigGen.createDumpLogs({reason: 'quitting through menu'}))
+        }
       }
+      // In case dump log doesn't exit for us
+      hideWindow()
+      setTimeout(() => {
+        quit()
+      }, 2000)
+    },
+    refreshUserFileEdits: throttle(() => dispatch(FsGen.createUserFileEditsLoad()), 1000 * 5),
+    showBug: () => {
+      const version = __VERSION__
+      SafeElectron.getShell().openExternal(
+        `https://github.com/keybase/client/issues/new?body=Keybase%20GUI%20Version:%20${encodeURIComponent(
+          version
+        )}`
+      )
+    },
+    showHelp: () => {
+      const link = urlHelper('help')
+      link && openUrl(link)
+      hideWindow()
+    },
+    showInFinder: () => dispatch(FsGen.createOpenPathInSystemFileManager({path: FsConstants.defaultPath})),
+    updateNow: isWindows || isDarwin ? () => dispatch(ConfigGen.createUpdateNow()) : undefined,
+    waitForKbfsDaemon: throttle(() => dispatch(FsGen.createWaitForKbfsDaemon()), 1000 * 5),
+  }),
+  (stateProps, dispatchProps, _ownProps: {}) => {
+    const {_showUser, _onRetrySync, ...dRest} = dispatchProps
+    const {config, ...rest} = stateProps
+    const {username} = config
+    return {
+      ...rest,
+      ...config,
+      ...dRest,
+      onRetrySync: () => dispatchProps._onRetrySync(username),
+      showUser: () => dispatchProps._showUser(username),
     }
-    // In case dump log doesn't exit for us
-    hideWindow()
-    setTimeout(() => {
-      quit()
-    }, 2000)
-  },
-  refreshUserFileEdits: throttle(() => dispatch(FsGen.createUserFileEditsLoad()), 1000 * 5),
-  showBug: () => {
-    const version = __VERSION__ // eslint-disable-line no-undef
-    SafeElectron.getShell().openExternal(
-      `https://github.com/keybase/client/issues/new?body=Keybase%20GUI%20Version:%20${encodeURIComponent(
-        version
-      )}`
-    )
-  },
-  showHelp: () => {
-    const link = urlHelper('help')
-    link && openUrl(link)
-    hideWindow()
-  },
-  showInFinder: () => dispatch(FsGen.createOpenPathInSystemFileManager({path: FsConstants.defaultPath})),
-  updateNow: isWindows || isDarwin ? () => dispatch(ConfigGen.createUpdateNow()) : undefined,
-  waitForKbfsDaemon: throttle(() => dispatch(FsGen.createWaitForKbfsDaemon()), 1000 * 5),
-})
-
-const mergeProps = (stateProps, dispatchProps, ownProps) => {
-  return {
-    ...stateProps,
-    ...dispatchProps,
-    onRetrySync: dispatchProps._onRetrySync(stateProps.username),
```

Comment:
actual bug!

Reply:
I don't think this was an actual bug? you mean the missing `() =>`? It's not the way we usually do things but the definition in `dispatchProps` was `(name) => () => ...`